framework/src/base/Moose.C

```diff
@@ -111,6 +111,7 @@ addActionTypes(Syntax & syntax)
   registerMooseObjectTask("add_variable",                 MooseVariableBase,      false);
   registerMooseObjectTask("add_aux_variable",             MooseVariableBase,      false);
   registerMooseObjectTask("add_elemental_field_variable", MooseVariableBase,      false);
+  registerMooseObjectTask("add_mortar_variable",          MooseVariableBase,      false);
```

Comment:
I don't think this line should be here (it only applies to stuff in the `[Variables]` block if I'm not mistaken).

Reply:
That is correct. Thanks for pointing this out!